### ISSUE TITLE
Custom Fileds Bugs

### ISSUE THREAD

**Issue by rudashi** [ThemeAvenue/Awesome-Support#110]
Currently Custom Fields don't support $placeholder.

This is the easy to repair, just Add in line 82 ( `WPAS_Custom_Fields_Display->text()` )
```
<?php if ( $field['args']['placeholder'] !== '' ): ?>placeholder="<?php echo $field['args']['placeholder'];?>"<?php endif; ?> 
```

`$desc` give me a lot of errors with `wp_kses()` when I add this arg to taxonomy type field.
Another weird error is, when I submit a ticket, value of Taxonomy is not saved.
But I'm able to edit ticket as Admin and add value from Taxonomy.

Any help?

PS: Sorry for my english.

**Comment by julien731** [comment]
Thanks for reporting this. Could you confirm which version of the plugin you're running? There was a problem with taxonomies not being saved properly in v3.1.4.

**Comment by rudashi** [comment]
I'm using last version (3.1.6)

**Comment by julien731** [comment]
Fixed in 03cb7e4e244d4c154767f75d48df364b9c7892c1
